http-service/src/main/java/net/runelite/http/service/chat/ChatController.java

```diff
@@ -100,4 +101,26 @@ public int getKc(@RequestParam String name)
 		}
 		return kc;
 	}
+
+	@PostMapping("/task")
+	public void submitTask(@RequestParam String name, @RequestParam String task, @RequestParam int amount, @RequestParam int initialAmount, @RequestParam String location)
+	{
+		if (task == null)
+		{
+			return;
+		}
+
+		chatService.setTask(name, task, amount, initialAmount, location);
+	}
+
+	@GetMapping("/task")
+	public String getTask(@RequestParam String name)
+	{
+		Map<String, String> properties = chatService.getTask(name);
+		if (properties == null || properties.get("task") == null)
+		{
+			throw new NotFoundException();
+		}
+		return properties.get("task") + "." + properties.get("amount") + "." + properties.get("initialAmount") + "." + properties.get("location");
```

Comment:
I doubt this is the right way to go about this, but I couldn't think of any way to return this information in the format of a `String`.